src/tests/ftest/pool/pda.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/python
+'''
+  (C) Copyright 2022 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+'''
+
+from apricot import TestWithServers
+
+class PoolPDAProperty(TestWithServers):
+    # pylint: disable=too-many-ancestors
+    # pylint: disable=too-few-public-methods
+    """run tests with pool pda property
+
+    Test Class Description: To validate pool pda works properly
+
+    :avocado: recursive
+    """
+    def verify_cont_pda(self, expected_value, match_str):
+        """
+        Verify the container pda property.
+
+        Args:
+            expected_value (int): expected container pda value
+        """
+        daos_cmd = self.get_daos_command()
+        cont_prop = daos_cmd.container_get_prop(self.pool.uuid,
+                                                self.container.uuid)
+        cont_prop_stdout = cont_prop.stdout_text
+        prop_list = cont_prop_stdout.split('\n')[1:]
+        cont_index = [i for i, word in enumerate(prop_list)
+                      if word.startswith(match_str)][0]
+        pda_str = (prop_list[cont_index].split(match_str)[1].strip())
+        pda_value = int(pda_str)
```

Comment:
Not necessary, but if you wanted to use a regular expression, this works, where `group(1)` corresponds to the group `([\d]+)` - sequence of digits -  in the pattern.
```
int(re.search("Performance domain affinity level of EC.*([\d]+)", cont_prop_stdout).group(1))
```

Reply:
I tested locally, it works, re-pushed with this good cleanup idea plus test PR tags, thanks!